src/components/table.js

```diff
@@ -108,29 +108,25 @@ export const paginator = props => {
   ])
 }
 
-const cellStyles = {
+const cellStyles = (col, total) => ({
   display: 'flex',
   alignItems: 'center',
-  paddingLeft: '1rem',
-  paddingRight: '1rem'
-}
+  paddingLeft: col === 0 ? '1.5rem' : '1rem',
+  paddingRight: col === total - 1 ? '1.5rem' : '1rem'
+})
 
 const styles = {
   cell: (col, total) => ({
-    ...cellStyles,
+    ...cellStyles(col, total),
     borderBottom: `1px solid ${colors.dark(0.2)}`,
-    borderLeft: `1px solid ${colors.dark(0.2)}`,
-    borderRight: col === total - 1 ? `1px solid ${colors.dark(0.2)}` : undefined
+    borderLeft: col === 0 ? undefined : `1px solid ${colors.dark(0.2)}`
```

Comment:
TOL: For me, this has a couple impacts: On the pages where the table butts up against the edge of the screen (data, jobs), it makes me think there's content off screen, and I keep wanting to scroll to it. And in the places where it doesn't (modal, method config), it looks like a style bug. Thoughts?

Reply:
Yeah, I definitely think this is worth discussing with the team and Lou. I'm thinking of this PR as a design tweak proposal.

Reply:
I agree with Brett. There are definitely areas where it needs work. To me, expanding tables like this shifts the paradigm from an application with scrollable panes to a scrollable document with floating tool/navigation areas.